ui/widgets/tooltip.js

```diff
@@ -140,6 +140,8 @@ $.widget( "ui.tooltip", {
 		}
 	},
 
+	_setOptionDisabled: $.noop,
```

Comment:
Oh, the style changes it's preventing are from the base widget... So, we should still be able to do what I said, the `._setOptionDisabled()` method in tooltip just wouldn't call `._super()`.